file_operations/build.gradle

```diff
@@ -21,6 +21,7 @@ android {
         release {
             minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            ndk.debugSymbolLevel = 'FULL'
```

Comment:
What's the size difference of bundle? I thought changing it to full increases the size with quite some margin.

Reply:
> I thought changing it to full increases the size with quite some margin.

There is just a single function in c. I will check though.

Reply:
[control.zip](https://github.com/TeamAmaze/AmazeFileManager/files/7608661/control.zip)
![imagen](https://user-images.githubusercontent.com/10991116/143581214-98c9ab75-e4bc-4dec-a55d-57acccd90b50.png)


[symbols.zip](https://github.com/TeamAmaze/AmazeFileManager/files/7608685/symbols.zip)

![imagen](https://user-images.githubusercontent.com/10991116/143581249-7640da15-e8d3-479e-9dde-5e86888ab991.png)